### ISSUE TITLE
Learning rate finder broke on master

### ISSUE THREAD

**Issue by borisdayma** [PyTorchLightning/pytorch-lightning#6471]
## 🐛 Bug

Learning rate finder stopped working on master branch ([latest commit tested](https://github.com/PyTorchLightning/pytorch-lightning/commit/d1db604c61edb3142960346fbcbbfe74250baeb7)).

### To Reproduce

*Note: the BoringModel was not running so I used an other model from the docs*

See [this colab](https://colab.research.google.com/drive/1U2-S_5g1Crh-pj2JqFAPVO5nvb8xu5Dl?usp=sharing) to reproduce the bug.
It works on latest released version (1.2.3) but not when installing from latest master branch (it diverges).

### Expected behavior

Learning rate finder should converge

**Comment by edenlightning** [comment]
@SkafteNicki mind taking a look?

**Comment by borisdayma** [comment]
Just for info, it now runs on master but the value of the learning rate is much higher than what it should be (`208929613.0854041` vs about `3` on last released version as of today).
